src/lib/select/select.ts

```diff
@@ -775,6 +778,17 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     return this._parentFormField ? `mat-${this._parentFormField.color}` : '';
   }
 
+  // TODO(josephperrott): Remove after 2018 spec updates are fully merged.
+  /** Sets the psuedo checkbox padding size based on the width of the psuedo checkbox. */
+  private _setPsuedoCheckboxPaddingSize() {
+    if (!SELECT_MULTIPLE_PANEL_PADDING_X) {
```

Comment:
It would be pretty easy to also gate this on `MatSelect.multiple` so we don't have to hit the DOM at all.